### DISCUSSION TITLE
Change frame in decoded picture buffer (DPB)

### DISCUSSION THREAD

**Discussion by cheptil** [fraunhoferhhi/vvenc#277] [Q&A]
Sorry if I double Issue 276

I try to use frames from DPB to make interpolated frame and put it back to DPB, for example I want to add such interpolated frame in the end of ref list - how can I do it in a most easy way?

**Comment by adamjw24** [comment]
See #276

**Comment by cheptil** [comment]
Sorry if I'm taking up a lot of your time

I understand how to dump original frame from pic structure, getOrigBuf method and stride - such a way I see original frame

But if I do the same with getRecoBuf method I see some noise, not recuntructed frame - how it can be, may be I dump reconstructed frame in a wrong place (in the end of EncGOP::xEncodePicture) ?
